test/parallel/test-stream-manualstart.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+const common = require('../common');
+const assert = require('assert');
+const fs = require('fs');
+
+fs.promises.open(__filename).then(common.mustCall((fd) => {
```

Comment:
why are you using `fs.promises`? Could you please use `fs`?

Reply:
Because a cook who doesn't eat its own food is not a good cook :smile: